Research/QuantBook.cs

```diff
@@ -324,7 +324,16 @@ public OptionHistory GetOptionHistory(Symbol symbol, DateTime start, DateTime? e
             // Load a canonical option Symbol if the user provides us with an underlying Symbol
             if (symbol.SecurityType != SecurityType.Option && symbol.SecurityType != SecurityType.FutureOption)
             {
-                symbol = AddOption(symbol, resolution, symbol.ID.Market).Symbol;
+                var option = AddOption(symbol, resolution, symbol.ID.Market);
+
+                // Allow 20 strikes from the money for futures. No expiry filter is applied
+                // so that any future contract provided will have data returned.
+                if (symbol.SecurityType == SecurityType.Future && !symbol.IsCanonical())
```

Comment:
If the provided symbol is a canonical future we should probably throw an exception?